R/visualization/plotBestMethodsATAC.R

```diff
@@ -67,7 +60,7 @@ plotBestMethodsATAC<- function(csv_atac_small_path, csv_atac_large_path){
     methods_name <- capitalize(methods_name)
     methods_name <- plyr::mapvalues(methods_name, 
                                     from = c("Seurat", "Seuratrpca", "Mnn", "Bbknn", "Trvae", "Scvi", "Liger", "Combat", "Saucie", "Fastmnn", "Desc", "Scanvi", "Scgen"), 
-                                    to = c("Seurat v3 CCA", "Seurat v3 RPCA", "MNN", "BBKNN", "trVAE", "scVI", "LIGER", "ComBat", "SAUCIE", "fastMNN", "DESC", "scANVI", "scGen"))
+                                    to = c("Seurat v3 CCA", "Seurat v3 RPCA", "MNN", "BBKNN", "trVAE", "scVI", "LIGER", "ComBat", "SAUCIE", "fastMNN", "DESC", "scANVI*", "scGen*"))
```

Comment:
I think the `*` is for the methods that need annotations? Just checking so I can do the same thing on the website.

Reply:
yes exactly, Malte asked me to add this * cause those would be the methods that use gene info

Reply:
cell type info, but yes :)

Reply:
👍 I'll try and do the same on the website then for consistency.